File: test_login.dart
import 'package:http/http.dart' as http;
import 'dart:convert';

void main() async {
  final response = await http.post(
    Uri.parse('http://169.239.251.102:280/~chika.amanna/Glaucoma_Detect/backend/auth.php?action=login'),
    body: json.encode({
      "email": "alice.green@glaucoma.com",
      "password": "doctor123",
    }),
  );
  print(response.body);
}

File: test_login.php
<?php
$ch = curl_init('http://169.239.251.102:280/~chika.amanna/Glaucoma_Detect/backend/auth.php?action=login');
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_POST, true);
curl_setopt($ch, CURLOPT_POSTFIELDS, json_encode([
    'email' => 'alice.green@glaucoma.com',
    'password' => 'doctor123'
]));
$response = curl_exec($ch);
echo "Response: " . $response . "\n";
?>
